plugins/channelrx/demodpager/pagerdemodsink.cpp

```diff
@@ -30,7 +30,6 @@
 
 PagerDemodSink::PagerDemodSink(PagerDemod *pagerDemod) :
         m_scopeSink(nullptr),
-        m_pagerDemod(pagerDemod),
```

Comment:
Remove from constructor parameters if not needed

Reply:
ok